packages/yoroi-extension/app/components/wallet/send/WalletSendFormRevamp.js

```diff
@@ -0,0 +1,682 @@
+import ReactToolboxMobxForm from '../../../utils/ReactToolboxMobxForm';
+import vjf from 'mobx-react-form/lib/validators/VJF';
+import { AmountInputRevamp } from '../../common/NumericInputRP';
+import styles from './WalletSendFormRevamp.scss';
+import globalMessages, { memoMessages, } from '../../../i18n/global-messages';
+import type { UriParams } from '../../../utils/URIHandling';
+import { getAddressPayload, isValidReceiveAddress } from '../../../api/ada/lib/storage/bridge/utils';
+import { MAX_MEMO_SIZE } from '../../../config/externalStorageConfig';
+import type { TokenRow, NetworkRow } from '../../../api/ada/lib/storage/database/primitives/tables';
+import {
+  formattedAmountToBigNumber,
+  formattedAmountToNaturalUnits,
+  truncateToken,
+} from '../../../utils/formatters';
+import config from '../../../config';
+import LocalizableError from '../../../i18n/LocalizableError';
+import type { $npm$ReactIntl$IntlFormat } from 'react-intl';
+import {
+  getTokenName, genFormatTokenAmount,
+  // getTokenStrictName, getTokenIdentifierIfExists,
+} from '../../../stores/stateless/tokenHelpers';
+import {
+  MultiToken,
+} from '../../../api/common/lib/MultiToken';
+import type {
+  TokenEntry,
+  TokenLookupKey,
+} from '../../../api/common/lib/MultiToken';
+import BigNumber from 'bignumber.js';
+import classnames from 'classnames';
+import SendFormHeader from './SendFormHeader';
+import { SEND_FORM_STEP } from '../../../types/WalletSendTypes';
+import { isErgo } from '../../../api/ada/lib/storage/database/prepackaged/networks';
+
+const messages = defineMessages({
+  receiverLabel: {
+    id: 'wallet.send.form.receiver.label',
+    defaultMessage: '!!!Receiver',
+  },
+  receiverHint: {
+    id: 'wallet.send.form.receiver.hint',
+    defaultMessage: '!!!Wallet Address',
+  },
+  dropdownAmountLabel: {
+    id: 'wallet.send.form.sendAll.dropdownAmountLabel',
+    defaultMessage: '!!!Send all {coinName}',
+  },
+  allTokens: {
+    id: 'wallet.send.form.sendAll.allTokens',
+    defaultMessage: '!!! + all tokens',
+  },
+  selectedAmountLable: {
+    id: 'wallet.send.form.sendAll.selectedAmountLable',
+    defaultMessage: '!!!Amount Options',
+  },
+  customAmount: {
+    id: 'wallet.send.form.sendAll.customAmount',
+    defaultMessage: '!!!Custom Amount',
+  },
+  transactionFeeError: {
+    id: 'wallet.send.form.transactionFeeError',
+    defaultMessage: '!!!Not enough Ada for fees. Try sending a smaller amount.',
+  },
+  calculatingFee: {
+    id: 'wallet.send.form.calculatingFee',
+    defaultMessage: '!!!Calculating fee...',
+  },
+  memoInvalidOptional: {
+    id: 'wallet.transaction.memo.optional.invalid',
+    defaultMessage: '!!!Memo cannot be more than {maxMemo} characters.',
+  },
+  willSendAll: {
+    id: 'wallet.send.form.willSendAll',
+    defaultMessage: '!!!Will Send All Tokens!'
+  },
+  amountMinAdaIncluded: {
+    id: 'wallet.send.form.revamp.amountMinAdaIncluded',
+    defaultMessage: '!!!Amount (includes min-ADA)'
```

Comment:
what does "(includes min-ADA)" mean?

Reply:
@Marta-Ryzhok can you explain? 

<img width="704" alt="Screen Shot 2022-01-30 at 3 41 59 AM" src="https://user-images.githubusercontent.com/72753578/151683457-4033747e-0399-42eb-95de-91c51de0a93a.png">